vtex/ab-tester
# Math tools to create confidence intervals from data

#### What is the purpose of this pull request?
This is the first part of two that should add the capability of calculate confidence intervals for certain metrics, given the data got during a AB test.

#### How should this be manually tested?
As this don't add any feature that affect any API, the bets way to test it is by linking this app with this change and also a log of the calculated value by these function added in this PR. Then compare the returned value with the expected.

#### Types of changes
